lib/active_model_serializers/adapter/json.rb

```diff
@@ -3,6 +3,7 @@ module Adapter
     class Json < Base
       def serializable_hash(options = nil)
         options = serialization_options(options)
+        instance_options[:fieldset] = fieldset
```

Comment:
why is this here and not in the base adapter? 

why in the attributes adapter is it `options[:fields] ||= fields || instance_options[:fields]` both adapters do basically the same thing, no?